### PR TITLE
Add new desktop page plus some text fixes

### LINKED ISSUE #21: Deploy And Maintain Branded Linux Clients (Enterprise Only)
Seems my previous contribution got lost with any of the docs migrations… Here's the only trace I was able to find:
https://github.com/owncloud/documentation/blob/9594137c5fadf348fe1a8701d45f6365ebbaec4c/admin_manual/enterprise_clients/deploy_branded_linux_clients.rst

@mmattel @EParzefall could you add it here?
https://doc.owncloud.com/branded_clients/next/branded_desktop_client/

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: https://github.com/owncloud/docs-client-branding/issues/21 (Deploy And Maintain Branded Linux Clients (Enterprise Only))

* Add a new document `branded_linux_desktop_clients.adoc`
* Fixed a bunch of text related stuff in the other desktop documents (could not resist after looking at it)
* Improved readability of the other desktop documents
* Replaced old links with new ones (owncloud.org does not exist anymore)

@michaelstingl 
Note the `branded_linux_desktop_clients.adoc` document may need a closer look on as it originally comes from an archived repo written in markdown and may contain content that is outdated like the far outdated OS releases of Ubuntu (14/15) or Fedora ect. If there is no change needed, drop a note that this is ok so.

A local build renders fine.

### CODE REVIEW

File: modules/ROOT/pages/branded_desktop_client/branded_desktop_client.adoc [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+In the ownBrander wizard at your account, start with the Common section at the top, and enter information common to all clients that you can build with ownBrander. You may override any settings inside the _Common_ section of the _Client_ sections.
+
+Then go to the _Desktop client section_ of ownBrander, which has two sections, _Required_ and _Optional_.
 
 image:branded_desktop_app/ownbrander-2.png[ownBrander screen for desktop client.]
 
-Work your way through the wizard, entering required elements and any optional elements you wish.
-When you have completed the wizard, press the *Generate Desktop Client* button.
-You will either get messages warning you of any items that need to be corrected, or a success message:
+Work your way through the wizard, enter required elements and any optional elements you wish. When you have completed the wizard, press the btn:[Generate Desktop Client] button. You will either get messages warning of any items that need to be corrected, or a success message:
```
Comment (phil-davis) [inline modules/ROOT/pages/branded_desktop_client/branded_desktop_client.adoc]:
```suggestion
Work your way through the wizard, enter required elements and any optional elements you wish. When you have completed the wizard, press the btn:[Generate Desktop Client] button. You will either get messages warning of any items that need to be corrected, or a success message.
```

File: modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc [hunk 2]
```diff
@@ -0,0 +1,121 @@
+= Deploy And Update Branded Linux Desktop Clients
+
+== Introduction
+
+As an ownBrander user, you can enable the build of branded Linux in the “Desktop” section of their account. You can download a `*.tar` file which contains everything to set up complete self-hosted Linux repositories
```
Comment (phil-davis) [inline modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc]:
```suggestion
As an ownBrander user, you can enable the build of branded Linux in the “Desktop” section of your account. You can download a `*.tar` file which contains everything to set up complete self-hosted Linux repositories
```

I am not so familiar with this stuff. Not sure who is "their" in "their account".

File: modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+│   ├── openSUSE_13.2
+│   │   └── mycloud-client.ymp
+│   └── openSUSE_ymp.html
+├── openSUSE_13.1
+│   └── …
+└── openSUSE_13.2
+└── …
+----
+
+The download folder provides detailed instructions for your users at `download/index.html` how to install the branded Linux clients on the selected Linux distributions. All location information in the HTML files is set to `download.example.com`. All metadata in the repo is set to `download.example.com` too.
```
Comment (phil-davis) [inline modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc]:
```suggestion
The download folder provides detailed instructions for your users at `download/index.html` about how to install the branded Linux clients on the selected Linux distributions. All location information in the HTML files is set to `download.example.com`. All metadata in the repo is set to `download.example.com` too.
```

File: modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  -p '.*-client' \
+  -i 'index.html' \
+  -f ..
+----
+
+Replace `\http://download.example.com/repo` in the above example with the base URL of your repository and save the file with a new name. (`download/mycloud.sh`)
+
+Then execute the script and check `download/index.html` on your webserver.
+
+The ownBrander build system can be configured to include the custom repo URL information in the `*.tar` file. Currently, this parameter isn’t available in the ownBrander web UI, but the ownCloud support (support@owncloud.com) can add this information to your ownBrander account.
```
Comment (phil-davis) [inline modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc]:
```suggestion
The ownBrander build system can be configured to include the custom repo URL information in the `*.tar` file. Currently, this parameter isn’t available in the ownBrander web UI, but the ownCloud support team (support@owncloud.com) can add this information to your ownBrander account.
```

Either "the ownCloud support team" or "ownCloud support" reads OK.

**Comment by phil-davis** [comment]
I have done an English language and format review - it looks OK. I am not familiar with the actual detail of the topic. So someone else can review the content for technical correctness.

**Review by michaelstingl — APPROVED** [review]
I updated the Linux distros

File: modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+├── Fedora_35
+│   └── …
+├── Ubuntu_18.04
+│   └── …
+├── Ubuntu_20.04
+│   └── …
+├── Ubuntu_21.04
+││   └── …
+├── Ubuntu_21.10
+│   └── …
```
Comment (mmattel) [inline modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc]:
```suggestion
│   └── …
├── Ubuntu_21.04
│   └── …
├── Ubuntu_21.10
│   └── …
```
Remove extra `|`

**Pull request merged**